proto/invocation.proto

```diff
@@ -267,6 +267,9 @@ message InvocationQuery {
 
   // The pattern for the targets built (exact match). Ex: "//..."
   string pattern = 15;
+
+  // A plaintext tag for the targets built (exact match). Ex: "my-cool-tag"
+  string tag = 16;
```

Comment:
lame question: would it be a lot harder to support repeated tags here? or a comma separated string?

Reply:
not really, but i'm out tomorrow so you're getting it monday :P

Reply:
just kidding, love 2 write software, here it is.